projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk/internal/control/job/jupyter_sample_job/10_notebook.ipynb

```diff
@@ -0,0 +1,36 @@
+{
+ "cells": [],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.6"
+  },
+  "pycharm": {
+   "stem_cell": {
+    "cell_type": "raw",
+    "source": [
```

Comment:
The default notebook does not contain any sample code and configurations for vdk-ipython
Will adding the sample code and configurations be a separate effort?

Reply:
no, it will be part of this PR